src/test/java/seedu/address/logic/commands/ListCommandTest.java

```diff
@@ -1,8 +1,6 @@
 package seedu.address.logic.commands;
 
 import static seedu.address.logic.commands.CommandTestUtil.assertCommandSuccess;
-import static seedu.address.logic.commands.CommandTestUtil.showPersonAtIndex;
-import static seedu.address.testutil.TypicalIndexes.INDEX_FIRST_PERSON;
```

Comment:
not relevant to the PR but we're migrating from INDEX_FIRST_PERSON to INDEX_FIRST_TRANSACTION within the same TypicalIndexes class, just fyi in case later on you wanna implement more unit tests here